packages/polymath-ui/src/components/ErrorBoundary/ErrorBoundary.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+import React from 'react';
+// import Raven from 'raven-js';
+
+export default class ErrorBoundary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { hasError: false };
+  }
+
+  componentDidCatch(error, errorInfo) {
+    // Display fallback UI
+    this.setState({ hasError: true });
+    // Raven.captureException(error, { extra: errorInfo });
```

Comment:
Should we remove this commented code?

Reply:
I can. I left it because we are about to add support for Sentry in the next sprint (Raven = Sentry). Otherwise I agree I never leave stuff commented in general. What do you think?